### PR TITLE
Setting a bp in json should give correct error

### LINKED ISSUE #53: Setting breakpoint in package.js gives sourcemap not found error

### PR DESCRIPTION
**Pull request by matthewloring**
Fixes #53

### CODE REVIEW

**Comment by matthewloring** [comment]
@ofrobots PTAL.

File: lib/v8debugapi.js [hunk 1]
```diff
@@ -149,7 +150,8 @@ module.exports.create = function(logger_, config_, fileStats_) {
                 breakpoint.location.line = mappedPos.line;
               } catch (err) {
                 return setErrorStatusAndCallback(cb, breakpoint,
-                  StatusMessage.CONDITION, messages.SOURCE_MAP_READ_ERROR + err);
+                  StatusMessage.BREAKPOINT_CONDITION,
```
Comment (ofrobots) [inline lib/v8debugapi.js]: Ooh, this was a typo before?

Comment (ofrobots) [inline lib/v8debugapi.js]: I think `BREAKPOINT_SOURCE_LOCATION` might be a better choice here. WDYT?

Comment (matthewloring) [inline lib/v8debugapi.js]: Unfortunately it was :frown: `BREAKPOINT_SOURCE_LOCATION` sounds good to me!

Comment (ofrobots) [inline lib/v8debugapi.js]: In that case, it sounds like we are missing a test-case. Can you add a test-case for this and other error returns from this function?  separate PR is fine.

**Comment by ofrobots** [comment]
LGTM.

**Pull request merged**